legacy/src/app/entry.js

```diff
@@ -328,8 +328,7 @@ function unhideRSDLinks() {
   rsdLinks.forEach(link => link.classList.remove("u-hide"));
 }
 
-/* @ngInject */
-const displayTemplate = ($rootScope, $window, $http) => {
+const renderHeader = ($window, $http) => {
```

Comment:
This should be decorated with `/* @ngInject */`

ngAnnotate does try to automatically find params that need to be injected, and it must be working in this case, but it doesn't always, so best to be explicit.

Reply:
As discussed, this should be fine as the injection is set to `explicitOnly`.